src/Randomizer.SMZ3/.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using Randomizer.Shared;
+
+namespace Randomizer.SMZ3
+{
+    /// <summary>
+    /// Specifies the item categories for an item type.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Field,
+        Inherited = false, AllowMultiple = false)]
+    public sealed class ItemCategoryAttribute : Attribute
```

Comment:
This file doesn't have a file name?

Reply:
Ah, think I accidentally deleted the filename rather than the file. lol